src/Main.cpp

```diff
@@ -140,6 +141,8 @@ struct BlindfoldChangingSystem
 
 struct PyramidPlunder : public Game
 {
+	LevelManager* levelManager;
```

Comment:
Use `RawPtr<T>` from `Definitions.h` instead of `T*` if you need pointers. Same thing for any other type that has alias defined in `Definitions.h`.

Also, if variable has more than one word in the name, split words by underscore instead of using CamelCase notation (same thing with fields in LevelManager class).